### PR TITLE
[frontend] Fix  bad design in Knowledge of an Individual/Organization/System + overview of Narrative

### LINKED ISSUE #5383: Bad design in Knowledge of an Individual/Organization/System + overview of Narrative
Go to Entities > Individuals.
Go to the overview of an individual.
Go to the knowledge tab.
There is an unused space on the right: 
image

Same for Organizations and Individuals.

There is also a bad design in the overview of a narrative (last containers box is too high):
image

### PR DESCRIPTION
**Pull request by CelineSebe**
### Proposed changes

* quick fixes of bad design in Narrative/Individual/Organization/System

### Related issues

* #5383 

### Checklist

- [ ] I consider the submitted work as finished
- [ ] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: opencti-platform/opencti-front/src/private/components/entities/individuals/IndividualKnowledge.jsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ class IndividualKnowledgeComponent extends Component {
     const { individual, viewAs } = this.props;
     const link = `/dashboard/entities/individuals/${individual.id}/knowledge`;
     return (
-      <div style={{ paddingRight: viewAs === 'knowledge' ? 200 : 0 }}>
+      <>
```
Comment (Kedae) [inline opencti-platform/opencti-front/src/private/components/entities/individuals/IndividualKnowledge.jsx]: Useless empty fragment

File: opencti-platform/opencti-front/src/private/components/entities/organizations/OrganizationKnowledge.jsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ class OrganizationKnowledgeComponent extends Component {
     const { organization, viewAs } = this.props;
     const link = `/dashboard/entities/organizations/${organization.id}/knowledge`;
     return (
-      <div style={{ paddingRight: viewAs === 'knowledge' ? 200 : 0 }}>
+      <>
```
Comment (Kedae) [inline opencti-platform/opencti-front/src/private/components/entities/organizations/OrganizationKnowledge.jsx]: Same

File: opencti-platform/opencti-front/src/private/components/techniques/Narratives.tsx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+          'revoked',
+          'killChainPhases',
+          'name',
+        ]}
+      >
+        {queryRef && (
+          <React.Suspense
+            fallback={
+              <>
+                {Array(20)
```
Comment (Kedae) [inline opencti-platform/opencti-front/src/private/components/techniques/Narratives.tsx]: Can you make it one line please ?

File: opencti-platform/opencti-front/src/private/components/techniques/Narratives.tsx [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+    {
+      searchTerm: '',
+      sortBy: 'name',
+      orderAsc: true,
+      openExports: false,
+      filters: emptyFilterGroup,
+    },
+  );
+
+  const renderLines = () => {
```
Comment (Kedae) [inline opencti-platform/opencti-front/src/private/components/techniques/Narratives.tsx]: This function is useless, you can inline it all in the render

File: opencti-platform/opencti-front/src/private/components/entities/individuals/IndividualKnowledge.jsx (hunk 1 above)
Comment (CelineSebe) [reply opencti-platform/opencti-front/src/private/components/entities/individuals/IndividualKnowledge.jsx]: Thanks, done!

File: opencti-platform/opencti-front/src/private/components/techniques/Narratives.tsx (hunk 3 above)
Comment (CelineSebe) [reply opencti-platform/opencti-front/src/private/components/techniques/Narratives.tsx]: I'm not sure I've correctly made this change.

**Review by Kedae — APPROVED** [review]

**Pull request merged**